pom.xml

```diff
@@ -186,7 +186,13 @@
             <plugin>
                 <groupId>com.jcabi</groupId>
                 <artifactId>jcabi-ssl-maven-plugin</artifactId>
-                <version>0.7.22</version>
+                <!--
+                    I'm using 1.0-SNAPSHOT only in order to be able to build s3auth.
```

Comment:
Let's format this as a puzzle, so that it can be picked up and made into a separate ticket to be fixed in the future. It's as simple as starting the first line of the comment with `@todo <issue number>:<estimate>`. For example:

```xml
<!--
    @todo #90:30min I'm using 1.0-SNAPSHOT only...
   ...
--->
```

See http://www.yegor256.com/2009/03/04/pdd.html for an explanation of how it works.